node/cli/src/service.rs

```diff
@@ -66,6 +67,8 @@ impl<F> Default for NodeConfig<F> where F: substrate_service::ServiceFactory {
 construct_service_factory! {
 	struct Factory {
 		Block = Block,
+		ConsensusPair = AuraPair,
+		FinalityPair = GrandpaPair,
```

Comment:
again, not happy about this; i don't want to bake this stuff in; please mark for refactoring ASAP.